datastore/build.gradle

```diff
@@ -19,6 +20,12 @@ dependencies {
 	runtimeOnly 'com.h2database:h2'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	compile group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
+	implementation'io.github.resilience4j:resilience4j-spring-boot2:1.7.1'
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation 'org.springframework.boot:spring-boot-starter-aop'
+	compile('io.github.resilience4j:resilience4j-spring-boot2:1.7.1')
```

Comment:
you shouldnt need these 3 lines....

Reply:
Assuming this isnt a joke, VSCode wouldn't work without them

Reply:
im not joking at all. What if you took out the 3 `implementation` lines then?